test/provider_os_memory.cpp

```diff
@@ -261,7 +261,7 @@ TEST_P(umfProviderTest, get_name) {
 
 TEST_P(umfProviderTest, free_INVALID_POINTER) {
     umf_result_t umf_result =
-        umfMemoryProviderFree(provider.get(), INVALID_PTR, 0);
+        umfMemoryProviderFree(provider.get(), INVALID_PTR, page_plus_64);
```

Comment:
could you check here 2 cases (with 0 and !0 size)?

Reply:
Test added.
Done.